backend/src/v4/services/frontend.js

```diff
@@ -31,39 +31,37 @@ module.exports.createApp = function () {
 	const express = require("express");
 	const compression = require("compression");
 	const bodyParser = require("body-parser");
-	const compress = require("compression");
 	const favicon = require("serve-favicon");
 	const app = express();
 	const path = require("path");
 	const configRoute = require("../routes/config");
 	const cors = require("cors");
 
-	app.use(compression());
+	app.use(compression({ level: 9 }));
+	const publicDir =  __dirname + "/../../../../public";
+	app.use(favicon(`${publicDir}/assets/images/favicon.ico`));
```

Comment:
https://app.travis-ci.com/github/3drepo/3drepo.io/jobs/582251044#L694

This is breaking v5 test because we don't pull favicon from the public directory (and v5 test doesn't build frontend so that favicon doesn't exists)